atom/common/node_bindings.cc

```diff
@@ -239,8 +241,17 @@ void NodeBindings::UvRunOnce() {
   v8::MicrotasksScope script_scope(env->isolate(),
                                    v8::MicrotasksScope::kRunMicrotasks);
 
+  if (!is_browser_)
+    TRACE_EVENT_BEGIN1("devtools.timeline", "FunctionCall", "data",
```

Comment:
Does it only show up in the timeline if the `data` is specified? Or could `TRACE_EVENT_BEGIN0` be used instead?